File: steps/chat_messages_stream.py
from pydantic import BaseModel
 
class ChatMessage(BaseModel):
    id: str
    user_id: str
    message: str
    timestamp: str
 
config = {
    "name": "chatMessage",
    "schema": ChatMessage.model_json_schema(),
    "baseConfig": {"storageType": "default"}
}

File: steps/chat-messages.stream.js
const config = {
  name: 'chatMessage',
  schema: {
    type: 'object',
    properties: {
      id: { type: 'string' },
      userId: { type: 'string' },
      message: { type: 'string' },
      timestamp: { type: 'string' }
    },
    required: ['id', 'userId', 'message', 'timestamp']
  },
  baseConfig: {
    storageType: 'default'
  }
}
 
module.exports = { config }